.github/workflows/release.yml

```diff
@@ -84,20 +86,24 @@ jobs:
               generate_release_notes: true
             })
 
-      - name: Check commit messages for feat or break
```

Comment:
It's recommended to follow the RESTful architecture when defining job names and structures.